packages/elements-core/src/components/TryIt/TryIt.spec.tsx

```diff
@@ -646,10 +650,10 @@ describe('TryIt', () => {
 
       await waitFor(() => expect(fetchMock).toHaveBeenCalledTimes(1));
 
-      // disable mocking and send
-      userEvent.click(mockingButton);
-      enableItem = await screen.getByRole('menuitemcheckbox', { name: /enabled/i });
-      userEvent.click(enableItem);
+      // select regular server and send
+      userEvent.click(serversButton);
+      let server1 = screen.getByRole('menuitemradio', { name: /live server/i });
+      act(() => userEvent.click(server1));
```

Comment:
what is the rationale for `act` here? userEven wraps actions in `act` anyways, as far as I understand, so this should not be needed. 🤔

Reply:
Likely leftover, I'll check and remove if possible.